CalibCalorimetry/EcalTPGTools/plugins/EcalTPGParamBuilder.cc

```diff
@@ -173,10 +173,10 @@ EcalTPGParamBuilder::EcalTPGParamBuilder(edm::ParameterSet const& pSet)
   //depending on the saturation scale: Et_sat_EB_                                                                                                                                                                
   if(SFGVB_SpikeKillingThreshold_ == -1 || (SFGVB_SpikeKillingThreshold_ > Et_sat_EB_)) 
     SFGVB_SpikeKillingThreshold_ = 1023; //nokilling
-  else
+  else 
     SFGVB_SpikeKillingThreshold_ = int(SFGVB_SpikeKillingThreshold_ * 1024/Et_sat_EB_);    
-    edm::LogInfo("TopInfo") << "INFO:SPIKE KILLING THRESHOLD (ADC)=" << SFGVB_SpikeKillingThreshold_ << "\n";
-
+  edm::LogInfo("TopInfo") << "INFO:SPIKE KILLING THRESHOLD (ADC)=" << SFGVB_SpikeKillingThreshold_ << "\n";
+  
```

Comment:
hello @davidlange6
uncontroversial (and no change of behaviour)
yet please @argiro @depasse please take note